src/sync.js

```diff
@@ -701,7 +701,31 @@ SyncApi.prototype._syncFromCache = async function(savedSync) {
     };
 
     try {
-        await this._processSyncResponse(syncEventData, data);
+        await this._processSyncResponse(syncEventData, data, false);
+
+        const breadcrumbs = this.client.store.getAccountData("im.vector.setting.breadcrumbs");
+        const breadcrumbsRooms = breadcrumbs?.getContent().recent_rooms || [];
+
+        this.client.getRooms().forEach(room => {
+            const readReceipt = room.getAccountData("m.fully_read");
```

Comment:
this is the read marker rather than the read receipt and it's the read receipt that counts for clearing notifications so may want the other one